tests/validation/CMakeLists.txt

```diff
@@ -31,7 +31,8 @@ set(VALIDATION_CUDA_SOURCES
 
 if(NMC_VALIDATION_DATA_DIR)
     if ("${CMAKE_VERSION}" MATCHES "^3.[78].")
-        message(WARNING "CMake ${CMAKE_VERSION} has broken FindCUDA; omitting NMC_DATADIR define.")
+        # disable warning
+        #message(WARNING "CMake ${CMAKE_VERSION} has broken FindCUDA; omitting NMC_DATADIR define.")
     else()
```

Comment:
Why is this warning disabled? We still have the problem while we use this version of `FindCUDA`.